ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -57,6 +59,48 @@ objects:
       secret:
         secretName: ${JOB_NAME_SAFE}-cluster-profile
 
+    initContainers:
+    - name: share-installer-src
+      image: registry.svc.ci.openshift.org/${NAMESPACE}/pipeline:src
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+        mkdir /tmp/installer-src
+        cp -R . /tmp/installer-src
+
+    - name: bazel-build-smoke
+      image: quay.io/coreos/tectonic-builder:bazel-v0.3
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      - name: artifacts
+        # bazel needs to write to .cache
+        mountPath: /.cache
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+        echo "ls -al /tmp/src-installer"
+        ls -al /tmp/installer-src
+        # wait for source code to show up
+        if [[ ! -d /tmp/installer-src ]]; then
+          echo "Waiting for installer source code to show up ..."
+          sleep 10 & wait
+        fi
+        export USER=bazel && which bazel
+        cd /tmp/installer-src
+        mkdir /tmp/output
+        /usr/bin/bazel --output_base=/tmp/output build "$@" smoke_tests
```

Comment:
Can we drop the `"${@}"`?  This script isn't going to be called with arguments, is it?